src/pages/ProductListing/index.jsx

```diff
@@ -0,0 +1,90 @@
+                        </div>
+                        <div class="price-range">
+                            <input type="range" className='range-slider' min="0" max="10"/>
+                            <ul class="filter-range">
+                                <li>From: <span>$1</span></li>
+                                <li>To: <span>$1000</span></li>
+                            </ul>
+                        </div>
+                        <div class="sort-container">
+                            <h3>Sort By:</h3>
+                            <ul class="price-sort">
+                                <li>
+                                    <input type="radio" id="sort-h2l" name="sort-selector" />
+                                    <label for="sort-h2l">Price - High to Low</label>
+                                </li>
+                                <li>
+                                    <input type="radio" id="sort-l2h" name="sort-selector" />
+                                    <label for="sort-l2h">Price - Low to High</label>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="rate-container">
+                            <h3>Rating</h3>
+                            <ul class="product-rating">
+                                <li>
+                                    <input type="radio" id="4star" name="star-selector" />
+                                    <label for="4star">4 Star & above</label>
+                                </li>
+                                <li>
+                                    <input type="radio" id="3star" name="star-selector" />
+                                    <label for="3star">3 Star & above</label>
+                                </li>
+                                <li>
+                                    <input type="radio" id="2star" name="star-selector" />
+                                    <label for="2star">2 Star & above</label>
+                                </li>
+                                <li>
+                                    <input type="radio" id="1star" name="star-selector" />
+                                    <label for="1star">1 Star & above</label>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </aside>
+                <section class="main-right-section">
+                    <div class="listing-card-section">
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
+                        <VerticalCard/>
```

Comment:
You can map over the data to create multiple `<VerticalCard>` components

Reply:
I haven't made the backend yet. I'll map over after implementing mockbee.